kong/plugins/jwt/schema.lua

```diff
@@ -18,5 +26,20 @@ return {
     claims_to_verify = {type = "array", enum = {"exp", "nbf"}},
     anonymous = {type = "string", default = "", func = check_user},
     run_on_preflight = {type = "boolean", default = true},
+    maximum_expiration = {type = "number", default = -1, func = check_maximum_expiration_positive},
```

Comment:
I think a more common pattern is to treat `0` as the "indefinite expiration", and any value below zero as invalid (as if it were an unsigned integer). The semantics are somewhat more obvious (`-1` has a strong "error-related" connotation and also feels strange in a time-related value).

Reply:
you are right. fixed.